include/real_time_tools/thread.hpp

```diff
@@ -114,6 +115,12 @@ namespace real_time_tools {
      * a non real time operation.
      */
     bool block_memory_;
+
+    /**
+     * @brief Optimization level of the CPU. Set to 0 to get proper real-time performance.
```

Comment:
This value has nothing to do with `Optimization level of the CPU`. Such a thing as optimization levels does not exist on CPUs :)

When running the computer, the CPU (or parts of its cores) are put to sleep / slower speed to save power. Though saving power and keeping the CPU more cool is often desired, this also means the latency to respond to an event is higher (as the CPU needs to wake up first). The `cpu_dma_latency` property specifies the maximum latency that is desired by the system operator. Putting this value to zero means "give me zero latency response", meaning the CPU is not put into sleeping mode by the operating system, yielding maximum realtime response.

@mwuethri , this PR got merged already. Can you please fix this in a followup PR?

Reply:
yes, you are right this should be documented better, i fixed this and created a new pull request. 

i did not really know what this parameter means because the function set_cpu_dma_latency has zero documentation, could you or whoever wrote this add a doc?